components/ReplyConnection.js

```diff
@@ -99,14 +99,15 @@ export default class ReplyConnection extends React.PureComponent {
   renderAuthor = () => {
     const { replyConnection } = this.props;
     const replyVersion = replyConnection.getIn(['reply', 'versions', 0]);
-    const connectionAuthor = replyConnection.get('user');
-    const replyAuthor = replyVersion.get('user');
+    const connectionAuthor = replyConnection.get('user') || Map();
+    const replyAuthor = replyVersion.get('user') || Map();
 
-    const connectionAuthorName = connectionAuthor
-      ? connectionAuthor.get('name')
-      : '有人';
+    const connectionAuthorName = connectionAuthor.get('name') || '有人';
 
-    if (replyAuthor && connectionAuthor.get('id') !== replyAuthor.get('id')) {
+    if (
+      replyAuthor.get('name') &&
```

Comment:
not sure 
```js
if( !replyAuthor.isEmpty() ...
```
can be directly present more meaning for the empty check? 
or more complex? haha~

Reply:
We are showing names here, so the following situations should not pass as well:
1. `replyAuthor` has `name` but value is null
2. `replyAuthor` does not have `name`

Therefore simple empty check is not enough. We should check for what we really need to display here.

Reply:
make sense 👍